flake8_trio/runner.py

```diff
@@ -112,7 +125,7 @@ def __init__(self, options: Namespace, module: Module):
     def run(self) -> Iterable[Error]:
         if not self.visitors:
             return
-        for v in self.visitors:
+        for v in chain(self.utility_visitors, self.visitors):
```

Comment:
```suggestion
        for v in (*self.utility_visitors, *self.visitors):
```

Could also use `+` because they're both tuples, but the "anything iterable" syntax is visibly safe.

Reply:
Thought the `chain` was nice and explicit, but don't mind either variant. The plus solution might also be slower, no?